server/public/shared/i18n/i18n.go

```diff
@@ -33,6 +33,32 @@ var T TranslateFunc
 var TDefault TranslateFunc
 
 var locales = make(map[string]string)
+
+// supportedLocales is a hard-coded list of locales considered ready for production use. It must
+// be kept in sync with ../../../../webapp/channels/src/i18n/i18n.jsx.
+var supportedLocales = []string{
```

Comment:
Here's the list of supported locales matching the webapp.

Reply:
This would be a nice improvement, since I think it would also allow us to give enduser control over what a "supported" language is. I'll consider it for a separate improvement after the dust settles.